README.md

```diff
@@ -1,6 +1,6 @@
 # Rover Campaigns iOS SDK
 
-This is Rover Campaigns, the marketing campaigns automation SDK. Rover Campaigns augments our core Rover product, the Experiences designer, to drive mobile engagement and monetization with better mobile campaigns.
+This is Rover Campaigns, the marketing campaigns automation SDK from Rover. Rover Campaigns augments our core Rover Experiences product enable mobile engagement and monetization with better mobile campaigns.
```

Comment:
I think you're missing the word "to" somewhere...

> Rover Campaigns augments our core Rover Experiences product enable mobile engagement and monetization with better mobile campaigns.

Reply:
Also, not part of this change, but there is an incorrect URL.

> The SDK is 100% open-source and available on GitHub (https://github.com/RoverPlatform/rover-ios).

(should be `/rover-campaigns-ios`)

Reply:
Another typo, not part of this PR change:

> Note: if you are looking to include additionally include Experiences functionality